kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/channel/Channel.java

```diff
@@ -361,17 +291,6 @@ public void setScale(Number scale) {
         this.valueScale = scale;
     }
 
-    /**
-     * Specifies the offset to be applied to the channel value
-     *
-     * @param offset
-     *            a double value that specifies the offset to be applied to the channel value
-     * @since 2.3
-     */
-    public void setOffset(double offset) {
-        this.valueOffset = offset;
-    }
-
```

Comment:
I forgot to deprecate them in the past release, we can mantain them but i have to throw an exception in case scaleoffset is different type

Reply:
I changed my mind, since the class is not immutable a client could change first the scale and offset and then the value type and the scale/offset values. So I only restored methods and deprecated them.